extensions/Bilibili/src/dynamicFeed.tsx

```diff
@@ -1,24 +1,66 @@
 import { useDynamicFeed } from "./hooks";
 import { checkLogin, formatUrl } from "./utils";
-import { NoLoginView, Video, Post } from "./components";
-import { List, getPreferenceValues } from "@raycast/api";
+import { NoLoginView, Post, Video } from "./components";
+import { List } from "@raycast/api";
+import { useState } from "react";
 
-interface Preferences {
-  justShowVideos: boolean;
+type KindType = { id: string; name: string };
+
+function DrinkDropdown(props: { kindTypes: KindType[]; onKindTypeChange: (newValue: string) => void }) {
+  const { kindTypes, onKindTypeChange } = props;
+  return (
+    <List.Dropdown
+      tooltip="Select Drink Type"
+      storeValue={true}
+      onChange={(newValue) => {
+        onKindTypeChange(newValue);
+      }}
+    >
+      <List.Dropdown.Section title="Alcoholic Beverages">
+        {kindTypes.map((kindType) => (
+          <List.Dropdown.Item key={kindType.id} title={kindType.name} value={kindType.id} />
+        ))}
+      </List.Dropdown.Section>
+    </List.Dropdown>
+  );
 }
 
 export default function Command() {
   if (!checkLogin()) return <NoLoginView />;
-  const preference = getPreferenceValues<Preferences>();
 
+  const [filterType, setFilterType] = useState("");
   const { dynamicItems, isLoading } = useDynamicFeed();
 
   const videoTypes = ["DYNAMIC_TYPE_LIVE_RCMD", "DYNAMIC_TYPE_AV"];
 
+  const kindTypes: KindType[] = [
+    { id: "0", name: "全部" },
+    { id: "1", name: "未看过的视频" },
+    { id: "2", name: "看过的视频" },
+    { id: "3", name: "视频" },
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

Reply:
Okay